### DISCUSSION TITLE
Settings button not appearing in web interface

### DISCUSSION THREAD

**Discussion by claymcgovern** [karakeep-app/karakeep#661] [Q&A] [answered]
Everything appears to be working properly except I am not able to access settings in the web interface. The button doesn't even show. Without that I am not able to import bookmarks.

**Comment by kamtschatka** [comment]
the url is `<hoarder-url>/settings/info`. Are you maybe looking in the old location? the new location is in the top right, when you click on your user name/avatar icon

**Comment by claymcgovern** [comment]
The URL works fine, but that settings page is not available in either the User Settings or Admin Settings that appear when I click my user icon. There is not a "Settings" link.

**Comment by kamtschatka** [comment, marked as answer]
I am not sure which "Settings" page you are talking about then. Obviously you can see the User Settings and the Admin Settings page, there is no other "Settings" page then.
If you want to import bookmarks you go to "User Settings" and then to "Import/Export" on the left side. If that does not appear, you might be on an older version. the current version is 0.19

**Reply by claymcgovern** [reply]
Thanks for your help. All working now.
